src/test/java/com/github/polimi_mt_acg/back2school/utils/rest/HTTPServerManager.java

```diff
@@ -0,0 +1,27 @@
+package com.github.polimi_mt_acg.back2school.utils.rest;
+
+import com.github.polimi_mt_acg.back2school.utils.JacksonCustomMapper;
+import org.glassfish.grizzly.http.server.HttpServer;
+import org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory;
+import org.glassfish.jersey.jackson.JacksonFeature;
+import org.glassfish.jersey.server.ResourceConfig;
+
+import java.net.URI;
+
+public class HTTPServerManager {
+
+  public static HttpServer startServer(Class resourceClassName, String packages) {
```

Comment:
This should be:
```java
public static HttpServer startServer(Class resourceClassName, String... packages) {
    // ...
```
as the .packages() method lets you pass an array of Strings if you have multiple packages to include in your GrizzylyHttpServer deployment. This may sometimes be required (and at some point of my development it was) so I would keep that window open.

Reply:
Thanks a lot for the suggested improvement! It's totally reasonable and meaningful. :)
I've just changed the method to the more correct version in the last commit https://github.com/polimi-mt-acg/back2school/pull/11/commits/18ba45bf3636108f65226f90e64b1a43f49fbf0b of this PR.